pages/api_integration-clients/indexer_client.mdx

```diff
@@ -292,14 +292,14 @@ market_candles = market_candles_response.data['candles']
 <Tabs items={["TypeScript", "Python"]}>
 <Tab>
 ```typescript copy
-// address is the wallet address on dYdX chain
+// address is the wallet address on dYdX Chain
```

Comment:
Note this file is generated as part of our swagger, so we will need to make the change there or else this will be overwritten